README.markdown

```diff
@@ -76,6 +76,11 @@ In place of a version pointer like "0.10", you can use the special default alias
     nvm use stable
     nvm run unstable --version
 
+If you want to install a new version of Node.js and migrate npm packages from a previous version:
+
+    nvm install v0.10.40 --reinstall-packages-from=0.10.39
+    nvm install v0.12.7 --reinstall-packages-from=0.12.6
```

Comment:
Thanks! Could you also add `nvm install node --reinstall-packages-from=node`, and perhaps change this line to `nvm install v0.12.7 --reinstall-packages-from=0.12` to convey that the "from" version is read before installing the new one?